src/expressions.md

```diff
@@ -600,6 +600,12 @@ within this loop may exit out of this loop or return control to its head.
 See [break expressions](#break-expressions) and [continue
 expressions](#continue-expressions).
 
+The return value of a `loop` expression is `!`, which can be coerced to any
+type. However, if there is a `break` expression in the loop, the return value
+of the loop becomes the one of the value provided to the [break
+expression](#break-expressions) causing the loop to end, which is `()` by
```

Comment:
The comma there is improper English. I think it'd flow better with the information in its own sentence though. Perhaps "When the break expression has no return value it defaults to `()`."